Pipfile

```diff
@@ -7,9 +7,7 @@ name = "pypi"
 google-cloud-ndb = "*"
 semver = "*"
 pyyaml = "*"
-# Has to be locked to 1.5.0 because docker pulls down specific libgit2 version that only works with 1.5.0
-# TODO: Update Dockerfile to use newer libgit2 version
-pygit2 = "==1.5.0"
+pygit2 = "*"
```

Comment:
This should probably be `= "1.9.2"` to match the version used by the Pipfile in worker.

Reply:
This is the root Pipfile which will be used for dev environments (and eventually setup.py). We can't make any assumptions there.